docs/src/main/asciidoc/reactive-database-clients.adoc

```diff
@@ -3,14 +3,20 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
-= Quarkus - Reactive Postgres Client
+= Quarkus - Reactive Database Clients
```

Comment:
How about naming it `the Reactive SQL Client`, just to be the same as what the original one is.

Reply:
Good point @BillyYccc

Reply:
Done